adagrams/game.py

```diff
@@ -1,11 +1,125 @@
+import random
+
 def draw_letters():
-    pass
+    letters = {
```

Comment:
Thanks for the feedback! I just want to make sure I understand. While the value of letters should be constant at the start of this function (every time a hand is drawn, the same values should be in the dictionary), I want the values of the letters to change (to reflect how many more of each letter is left before the next draw in the hand) while the for loop is running. Is it still considered a constant, even though the object is modified while the for loop is running? In my head, the dictionary I created for the score_word function is a constant because it is meant to never (ever) change (so the variable should be LETTER_POINTS to reflect that it should never change). Am I thinking about this wrong?

Reply:
Oh, I messed up.  I thought that letters was a global variable that you were just using to reference info but now I see on line 39 that you are changing `letters` by decrementing the value. Therefore it's not a constant and doen't need to be named with all caps. 

Your understanding of a constant is correct!